### DISCUSSION TITLE
How to obtain the shot gather of elastic wave forward modeling through 06_elastic.ipynb

### DISCUSSION THREAD

**Discussion by shenghanlin** [devitocodes/devito#1671] [Q&A]
I'm wondering how to to obtain the shot gather of elastic wave forward modeling through 06_elastic.ipynb.
After we setting Reciver grid, what should we do next?

**Comment by mloubout** [comment]
You can check the full elastic operator at https://github.com/devitocodes/devito/blob/master/examples/seismic/elastic/operators.py for the full src/rec setup as well.
